plugins/modules/ipa_pwpolicy.py

```diff
@@ -64,6 +64,21 @@
     lockouttime:
         description: Period (in seconds) for which users are locked out.
         type: str
+    gracelimit:
+        description: Maximum number of ldap logins after password expiration.
+        type: str
```

Comment:
```suggestion
        type: str
        version_added: 8.2.0
```
Same for the other new options.

Reply:
Done